plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/BaseJdbcTypeMappingTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import io.trino.testing.AbstractTestQueryFramework;
+import org.testng.annotations.BeforeClass;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+
+import static com.google.common.base.Verify.verify;
+
+public abstract class BaseJdbcTypeMappingTest
+        extends AbstractTestQueryFramework
+{
+    protected static final LocalDateTime beforeEpoch = LocalDateTime.of(1958, 1, 1, 13, 18, 3, 123_000_000);
```

Comment:
These variables are not very usable with SqlDataTypeTest because it's literal based.

I'd argue to store a String here and convert to LocalDateTime in `verifyZones`.